cmd/oras/internal/option/remote_test.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package option_test
+
+import (
+	"context"
+	"crypto/x509"
+	"encoding/pem"
+	"os"
+	"path/filepath"
+	"testing"
+
+	nhttp "net/http"
+	"net/http/httptest"
+
+	"github.com/google/uuid"
+	"oras.land/oras-go/v2/registry/remote/auth"
+	"oras.land/oras/cmd/oras/internal/option"
+)
+
+var ts *httptest.Server
+
+func TestMain(m *testing.M) {
+	// Test server
+	ts = httptest.NewTLSServer(nhttp.HandlerFunc(func(w nhttp.ResponseWriter, r *nhttp.Request) {
+		p := r.URL.Path
+		m := r.Method
+		switch {
+		case p == "/v2/" && m == "GET":
+			w.WriteHeader(nhttp.StatusOK)
+		}
+	}))
+	defer ts.Close()
+	m.Run()
+}
+
+func Test_AuthClient_RawCredential(t *testing.T) {
+	want := auth.Credential{
+		Username: uuid.New().String(),
+		Password: uuid.New().String(),
```

Comment:
Do we really need the `uuid` package here as it introduced a new dependency?

Reply:
It's not necessary. I can write an function to generate random letter sequence for testing